src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -162,15 +173,31 @@ public static int MainInternal(string[] args, CommandOutputLogger log)
             return exitCode;
         }
 
-
-        private static CommandLineApplication InitializeApp(string[] args, CommandOutputLogger log)
+        private static Func<ILogger> GenerateLoggerHidePrefix(CommandOutputLogger log)
         {
-            // Many commands don't want prefixes output. Use this func instead of () => log to set the HidePrefix property first.
-            Func<ILogger> getHidePrefixLogger = () =>
+            return () =>
             {
                 log.HidePrefixForInfoAndMinimal = true;
                 return log;
             };
+        }
+
+        private static Command InitializeSystemCommandLineApp(string[] args, CommandOutputLogger log)
+        {
+            var app = new RootCommand();
+
+            if (!args.Any() || args[0] != "package")
```

Comment:
I don't understand what the `"package"` is here. It's not mentioned in the docs, https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-list-source

Reply:
I just copied if log from here:

https://github.com/NuGet/NuGet.Client/blob/1ad750307b4c227b78d3666fc68f8c71894b2f43/src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs#L180-L199